dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/webcontainerext/AbstractJSPExtensionServletWrapper.java

```diff
@@ -696,6 +676,26 @@ protected void createClassLoader() {
         }
     }
 
+    private static final PermissionCollection ALLPERMISSIONS;
+    static {
+        AllPermission allPerm = new AllPermission();
+        ALLPERMISSIONS = allPerm.newPermissionCollection();
+        if (ALLPERMISSIONS != null) {
+            ALLPERMISSIONS.add(allPerm);
+        }
+    }
+    private PermissionCollection createPermissionCollection0() throws MalformedURLException {
```

Comment:
Was the `0` at the end intentional?

Reply:
It was, but we can change it to `Impl` if you want or anything else that makes sense to you.  Other projects I work on use `0` as an internal impl detail for private methods.

Reply:
I'd lean toward Impl myself.

Reply:
Yes, it was to keep it separate from the createPermissionCollection() method.

Reply:
Impl it is.